pyteal/ast/abi/uint.py

```diff
@@ -219,8 +222,8 @@ def type_spec(self) -> UintTypeSpec:
     def get(self) -> Expr:
         return self.stored_value.load()
 
-    def set(self, value: Union[int, Expr, "Uint", ComputedType]) -> Expr:
-        if isinstance(value, ComputedType):
+    def set(self, value: Union[int, Expr, "Uint", ComputedValue[T]]) -> Expr:
```

Comment:
I think you'd want `self: T` as well so you can ensure the `ComputedValue` matches the instance type

Reply:
nice, do you think in `tuple.py` case, your suggestion applies?